fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/app/api/ServerApplication.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2020 Fraunhofer FOKUS and others. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contact: mosaic@fokus.fraunhofer.de
+ */
+
+package org.eclipse.mosaic.fed.application.app.api;
+
+import org.eclipse.mosaic.fed.application.app.api.os.ServerOperatingSystem;
+
+/**
+ * Note: This interface is empty for now and currently only functions as a marker-interface. Future extensions
+ * might add features.
+ */
+public interface ServerApplication extends Application, OperatingSystemAccess<ServerOperatingSystem> {
```

Comment:
Do we really need this kind of interface? These group of interfaces are usually only implemented optionally by the application to show that they are interested in certain methods to implement (e.g. receiveV2xMessage in CommunicationApplication). We could add this interface later if there's is any "event-hook" method which fits in the server application context.

Reply:
Removed.